presto-main/src/test/java/com/facebook/presto/sql/query/QueryAssertions.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql.query;
+
+import com.facebook.presto.Session;
+import com.facebook.presto.testing.LocalQueryRunner;
+import com.facebook.presto.testing.MaterializedResult;
+import com.facebook.presto.testing.MaterializedRow;
+import com.facebook.presto.testing.QueryRunner;
+import org.intellij.lang.annotations.Language;
+
+import java.util.List;
+
+import static com.facebook.presto.testing.TestingSession.testSessionBuilder;
+import static io.airlift.testing.Assertions.assertEqualsIgnoreOrder;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.fail;
+
+class QueryAssertions
```

Comment:
There is a `QueryAssertions` class already that contains static methods that accept `QueryRunner` as an argument.

Why can't you just use `AbstractTestQueryFramework` to be the base class for your tests? See `com.facebook.presto.connector.jmx.TestJmxQueries` for example.

Reply:
That's in presto-tests. This one is directly in presto-main and is intended to test just the engine. AbstractTestQueryFramework is used for testing connectors, the engine, etc, and it's become the kitchen sink of all tests.